app/src/main/java/de/bitshares_munich/database/SCWallDatabaseContract.java

```diff
@@ -49,4 +49,89 @@ private SCWallDatabaseContract(){}
         public static final String COLUMN_ID = "id";
         public static final String COLUMN_NAME = "name";
     }
+
+    //CrypotoCore
+
+    public static class Seeds implements BaseColumns {
+        public static final String TABLE_NAME = "seeds";
+        public static final String COLUMN_ID = "id";
+        public static final String COLUMN_TYPE = "derived_seed_type";
+        public static final String COLUMN_MNEMONIC = "mnemonic";
+        public static final String COLUMN_ADDITIONAL = "additional";
+    }
+
+    public static class GeneralAccounts implements BaseColumns {
+        public static final String TABLE_NAME = "general_accounts";
+        public static final String COLUMN_ID = "id";
+        public static final String COLUMN_NAME = "account_name"; //this is only to tag this account
+        public static final String COLUMN_TYPE = "coin_type";//bitcoin,litecoin
```

Comment:
I might suggest renaming this constant to COLUMN_COIN_TYPE to completely clarify its meaning and avoid confusion as there already seems to be another COLUMN_TYPE constant being used in another static class.